core/resources/scene.yaml

```diff
@@ -110,8 +122,13 @@ layers:
         draw:
             polygons:
                 order: 1
-                color: '#fffffa'
+                #color: '#fffffa'
+                color: global.color.colorA
                 interactive: true # currently ignored
+            lines:
+                order: 2
+                color: function() { return global.color.colorE; }
```

Comment:
I'm a little confused by the last comment, but I'll explain how I'm thinking about this now: 

When we evaluate the global properties block at scene load time, we'll create a global object called `global` in the duktape JS context. `global` will be populated with all of the members of the corresponding YAML block. While we are adding these members, we'll take any string that looks like a function and instead of giving `global` a JS string for that member, we'll give it a JS function (as evaluated by duktape). 

This supports things that can't be done with simple substitution, e.g.:
```yaml
global:
  convert_color: function(hsv) { ... }
...
  color: function() { return global.convert_color(feature.color); }
```